MM-28803 - Update Maximum Users Per Team (#4157)
mattermost/docs: source/cloud/cloud-administration/site-configuration.rst

There are seven different settings that allow you to control users and teams.

1. **Max Users Per Team:** The Max Users Per Team refers to the size of the *team site* which is a workspace a *team of people* inhabits. A team of people is considered a small organization where people work closely together towards a specific shared goal and share the same etiquette. In the physical world, a team of people could typically be seated around a single table to have a meal and discuss their project. The default maximum of 1000 people is at the extreme high end of a single team of people. At this point organizations are more often 'multiple teams of people' and investments in explicitly defining etiquette, such as channel organization or turning on policy features in Enterprise Edition, are often used to scale the high levels of productivity found in a team of people using Mattermost to multiple teams of people.
2. **Max Channels Per Team:** Set the maximum number of channels per team, including both active and deleted channels.

> 3. **Enable users to open Direct Message channels with:** You can configure if users can message any user on the Mattermost workspace or just users in the team that they are members of the teams the user is a member of. This setting adjusts the users returned in the Direct Messages More menu and CTRL/CMD+K channel switcher only lists users on the current team. This setting only affects the UI, not permissions on the server. For instance, a Direct Message channel can be created with anyone on the server regardless of this setting.

Reviewer: ```suggestion
3. **Enable users to open Direct Message channels with:** You can configure whether users can message any user on the Mattermost workspace or only users in the same team as them. This setting adjusts the users returned in the Direct Messages **More** menu and CTRL/CMD+K channel switcher only lists users on the current team. This setting only affects the UI, not permissions on the server. For instance, a Direct Message channel can be created with anyone on the server regardless of this setting.
```
